databricks/koalas/groupby.py

```diff
@@ -1396,7 +1397,7 @@ def ffill(self, limit=None):
         2  NaN  NaN  5
         3  3.0  1.0  4
         """
-        return self._fillna(method='ffill', limit=limit)
+        return self._fillna(method='ffill', inplace=inplace, limit=limit)
```

Comment:
Can we add a test case for this one? I think it doesn't work.

Reply:
ah, you're right, and pandas doesn't support them.

- koalas
```python
>>> df.groupby(['A']).ffill(inplace=True).sort_index()
Traceback (most recent call last):
...
AssertionError: (3, 4)
```

- pandas
```python
>>> df.to_pandas().groupby(['A']).ffill(inplace=True).sort_index()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: pad() got an unexpected keyword argument 'inplace'
```

i'll revert those kinda change.